MSAL/src/native_auth/network/errors/MSALNativeAuthSubErrorCode.swift

```diff
@@ -24,7 +24,7 @@
 
 import Foundation
 
-enum MSALNativeAuthSubErrorCode: String, Decodable, CaseIterable, Equatable {
+enum MSALNativeAuthSubErrorCode: String, Decodable, CaseIterable, Equatable, MSALNativeAuthUnknownCaseProtocol {
```

Comment:
nit: Should we remove "CaseIterable" implementation from all enums declaration given that this protocol is already implemented by "MSALNativeAuthUnknownCaseProtocol" ?

Reply:
Done